t/job-manager/sched-helper.sh

```diff
@@ -43,7 +43,7 @@ jmgr_check_state() {
 _jmgr_get_annotation() {
         local id=$(flux job id $1)
         local key=$2
-        local note="$(${JMGR_JOB_LIST} | grep ${id} | cut -f 6- | jq ."${key}")"
+        local note="$(${JMGR_JOB_LIST} | grep ${id} | cut -f 7- | jq ."${key}")"
```

Comment:
This kind of thing in tests makes me cringe. Could we update the test job listing helper to optionally restrict output by key directly and get rid of the brittle `cut -f N-` used here? Anything to help a future test writer...

Reply:
yeah, i concur.  Let me see if it'd be a huge effort to redo that tool and all the tests associated with it.